lib/webtask.js

```diff
@@ -708,6 +756,42 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function validate_auth_token(secret, expected_aud) {
+            var auth_token;
+            var decoded_auth_token;
+            var userinfo;
+
+            console.log('\n\n\n\n');
+            console.log(secret);
+
+            try {
+                secret = new Buffer(secret, 'base64');
+            } catch(e) {
+                throw { code: 403, message: 'WEBTASK_JWT_SECRET must be base64 encoded: ' + e.message, req_id: req.req_id };
+            }
+
+            try {
+                auth_token = (req.query.key || req.headers['authorization']
+                    .match(/^bearer\s+(.+)$/i)[1]);
+            } catch(e) {
+                throw { code: 400, message: 'could not parse authorization header' + e.message, req_id: req.req_id };
```

Comment:
Can you be more descriptive here, mention the JWT token must be passed as a bearer token in the authorization header. In logs it would be good to capture what was actually passed.